crypto/sr25519/batch.go

```diff
@@ -3,40 +3,41 @@ package sr25519
 import (
 	"fmt"
 
-	schnorrkel "github.com/ChainSafe/go-schnorrkel"
+	"github.com/oasisprotocol/curve25519-voi/primitives/sr25519"
 
 	"github.com/tendermint/tendermint/crypto"
 )
 
-var _ crypto.BatchVerifier = BatchVerifier{}
```

Comment:
why remove?

Reply:
See line 192 (It could be moved up, but not implementing the interface will cause a build failure anyway).

Reply:
Guess it got lost in the rewrite, I can re-add it.  Though the ctor returning the interface is sufficient to cause a build failure.

Reply:
Re-added.